about.html

```diff
@@ -109,29 +115,31 @@ <h1>
       <div class="lastpart">
         <div class="held">
           <h2>See the past meetings held</h2>
-          <p id="past"></p>
+          <p id="past2"></p>
         </div>
         <p>
           Take a look at the last two Business meeetings <br />
           held in USA
         </p>
         <div class="containerimages">
           <div class="lastpartimg newyork">
-            <div id="redcolor">
+            <div id="redcolor2">
               <h1>2012</h1>
               <p id="ptag">Meeting in New York City</p>
             </div>
           </div>
           <div class="lastpartimg chicago">
             <div id="redcolor">
               <h1>2016</h1>
-              <p id="ptag">Meeting in Chicago</p>
+              <p id="ptag2">Meeting in Chicago</p>
             </div>
           </div>
         </div>
       </div>
 
       <!-- LAST CONTAINER -->
+    </section>
+    <footer>
       <div class="lastcontainer">
         <div class="partner">
           <h1>Partner</h1>
```

Comment:
- [ ] [OPTIONAL] Footer for mobile screens is not responsive, I strongly recommend you to fix the width: 

![image](https://user-images.githubusercontent.com/77038610/147339756-5c7ed8f2-9df8-4a5b-8fa0-c1eab022c16f.png)